randovania/game_description/pickup/pickup_entry.py

```diff
@@ -67,6 +67,7 @@ class PickupGeneratorParams:
     probability_offset: float = 0.0
     probability_multiplier: float = 1.0
     required_progression: int = 0
+    index_age_impact: float = 1.0  # by how much the PickupIndex's age is incremented when this pickup is placed
```

Comment:
Opening a thread here so that my testing results can get hidden easier when navigating on this PR

Reply:
attempt 1: see here https://github.com/randovania/randovania/pull/6607#issuecomment-2131307449

Reply:
attempt 2: change etank age impact and ammo age impact to 0.2
(left: all 1.0, right: etank and ammo 0.2)
![grafik](https://github.com/randovania/randovania/assets/38186597/bc5ca95a-91f9-4045-a5da-0f91ac840b6c)
artifacts seem more balanced around lower mines, but FHA and PMT are still outsiders. As was theorized earlier, their issue seems to be unrelated to this one.
Comparing the progression probability column is also interesting (yellow highlighted big changes):
![grafik](https://github.com/randovania/randovania/assets/38186597/0bfe0989-f9e7-4227-840c-e88ef4f78d3f)

Reply:
attempt 3: change etank and missiles to 0.1.
left: all 1.0, middle: etank and ammo 0.2, right: etank and ammo 0.1
artifacts:
![grafik](https://github.com/randovania/randovania/assets/38186597/e6df5fe8-87f8-4d46-80b4-bd3cdce198ea)


progression probability:
![grafik](https://github.com/randovania/randovania/assets/38186597/3befee15-e9b8-44b4-bd2a-c6cc67ba8519)

---

I thought the results would be worse than attempt 2, but at first glance, they seem nicer?

I had a look at phazon suit and it seems here a bit more evened out. But that could also just be within margin of error (still only 1k seeds)
![grafik](https://github.com/randovania/randovania/assets/38186597/cd643d4d-84a4-452d-b4e0-364137387a28)

I also had a look at the bosses which... are basically unchanged.
In fact, if I look at a table that shows me everything that has a bigger difference than 2%. The results are:
- Artifacts. Few changes in transport tallon B (tallon), life grove (tunnel), storage/security cave, early magmoor, plasma processing, ToL
- Etank placement. the only noteworthy one that ToL now has an 11% decrease in giving E
- missile expansions. Life grove tunnel, storage cave, chapel of the elders, lower mines noticeably have lower chance of giving missiles, ToL chance got bumped by ~14%
- phazon suit: ~3%decrease on OP
- progression probability: life grove, phen transport access, control tower, main quarry, magmoor workstation, elder chamber got noticeably bumped. research core, PCA, elite quarters got noticeably nerfed. ToL got decreased by 17.9%

---

I currently like attempt 3 the most. Unsure of what else should be tried. I could try lowering etanks even more in order to get them more into the ~14%ish range? Have the feeling that that's something else tho.

Reply:
I'm sending my games here off in the meantime + csv's if someone else wants to look at the data.
All starter preset, 1k games each.
1.0 value for everything:
[all-one.zip](https://github.com/randovania/randovania/files/15444352/all-one.zip)

0.25 for ammo only (aka this PR):
[ammo-025.zip](https://github.com/randovania/randovania/files/15444354/ammo-025.zip)

0.2 for ammo and etanks:
[etank-and-ammo-02.zip](https://github.com/randovania/randovania/files/15444355/etank-and-ammo-02.zip)

0.1 for ammo and etanks:
[01-for-etank-and-ammo.zip](https://github.com/randovania/randovania/files/15444358/01-for-etank-and-ammo.zip)

Reply:
Just for confirmation's sake, i tried setting etank and ammo to 4.0
![grafik](https://github.com/randovania/randovania/assets/38186597/b9ec140e-411a-480f-9ae3-4f8f533fc5b2)

Reply:
I find it easier to understand what's changed between builds by using conditional formatting on delta

![image](https://github.com/randovania/randovania/assets/14116556/2c9eb5d7-0dd8-4145-89f7-63f11664455b)

I'm worried this is just shifting the hotspot to a different part of the game, rather than fully diffusing the placement probability.

Honestly I thought the only way to fix this was to add a feature where game maintainers could pick some items at their discretion to get randomly pre-filled; this is a good close 2nd